example/javascriptlib/module/5-resources/build.mill

```diff
@@ -0,0 +1,30 @@
+package build
+
+import mill._, javascriptlib._
+import ujson._
+
+object foo extends TypeScriptModule {
+  object test extends TypeScriptTests with TestModule.Jest {
+    def mkENV = super.mkENV() + ("OTHER_FILES" -> (compile()._2.path / "other-files").toString)
+  }
+}
+
+// Documentation for mill.example.javascriptlib
+
+/** Usage
+> mill foo.run
+Hello World Resource File
+
+> mill foo.test
+PASS .../foo.test.ts
+...
+Test Suites:...1 passed, 1 total...
+Tests:...3 passed, 3 total...
+...
+
+> mill foo.bundle
+Build succeeded!
+
+> node out/foo/bundle.dest/bundle.js
+Hello World Resource File
```

Comment:
We should make sure this works even if we `rm -rf` the `out/` folder, just to ensure the bundle really does bundle all the resources necessary (as is the case for Java/Scala/Kotlin/Python)